camus-sweeper/src/main/java/com/linkedin/camus/sweeper/HiddenFilter.java

```diff
@@ -0,0 +1,17 @@
+package com.linkedin.camus.sweeper;
+
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.fs.PathFilter;
+
+public class HiddenFilter implements PathFilter {
+  @Override
+  public boolean accept(Path path) {
+    String name = path.getName();
+    if (name.startsWith("_") || name.startsWith(".")) {
+      return false;
+    }
+
+    return true;
+  }
```

Comment:
You can replace this with

return !(name.startsWith("_") || name.startsWith("."))

Also can you add java docs for this class